src/tests/sound_from_file_test.py

```diff
@@ -90,22 +91,18 @@ def visualize_d(d, sounds, test_sounds):
         print(f"{color}{r0.ljust(8)}\t{r1.ljust(8)}{bcolors.ENDC}")
 
 
-def move_sounds(sounds, coef):
+def try_to_move_sounds(sounds, coef):
     sound1 = []
     for s in sounds:
-        ds = s.rhythmic_value
+        ds = s.duration
         d = False
-        if ds.endswith('.'):
-            d = True
-            ds = ds[:-1]
-        try:
-            ds = str(int(int(ds)*coef))
-        except Exception:
-            ds = ""
-        if d:
-            ds = f"{ds}."
+        if ds not in constants.LEGAL_NOT_DOTTED_DURATION_VALUES:
+            ds = (ds//3)*2
+        ds = int(ds*coef)
+        if ds not in constants.LEGAL_DURATION_VALUES:
+            return None
```

Comment:
You missed the part with remembering the dot in `d=True` line. This few lines just multiplies note by coef (with respect to dotted notes).

Reply:
I see you fixed this

Reply:
I fixed